### DISCUSSION TITLE
exp（850）How do I solve this problem,

### DISCUSSION THREAD

**Discussion by ZEROSL99** [coin-or/Ipopt#743] [Q&A]
![image](https://github.com/coin-or/Ipopt/assets/52021625/a58094a3-fb8d-4154-ad00-a4c5d7b841c7)
![image](https://github.com/coin-or/Ipopt/assets/52021625/1cfffa53-3f4a-4dbc-9a0f-38c3b67fcf29)
Hello Because of the following formula, For D33 ,when (xr+12) > 0,  then yr>-∞,and that will cause the following solution to fail. For D11  when (xr+12) > 0,then  yr<∞,and that will also cause the following solution to fail.How do I solve this problem, or how do I set limits

![image](https://github.com/coin-or/Ipopt/assets/52021625/6e3e5245-d3e1-47f1-8b8e-64da048a469c)

**Comment by svigerske** [comment]
To avoid that xr+12 > 0, setting the upper bound of variable xr to -12 or lower could help.

**Reply by ZEROSL99** [reply]
I'm sorry, I didn't make that clear, but xr can be greater than and less than 12, but when xr+12> 0, you get yr> -∞, this is beyond the scope of evaluation, what should I do about it

**Reply by ZEROSL99** [reply]
For example, when y>5000, the computer can recognize 5000, but. -∞, the computer can not calculate or recognize the -∞ boundary,

**Reply by svigerske** [reply]
You may have to look into scaling your variables and equations such that values are in a reasonable range.

**Reply by ZEROSL99** [reply]
SO how to scale  variables and equations

**Reply by ZEROSL99** [reply]
Hello, I have scaled the function, but a point with a denominator of 0 appears, causing the following problem, what should I do? Is there any way to initialize or set or ignore this zero point separately?

**Comment by ZEROSL99** [comment]
Hello, I have scaled the function, but a point with a denominator of 0 appears, causing the following problem, what should I do? Is there any way to initialize or set or ignore this zero point separately?

![image](https://github.com/coin-or/Ipopt/assets/52021625/a313104e-aa38-4d58-b2e1-a7688fe653b9)
![image](https://github.com/coin-or/Ipopt/assets/52021625/941492a5-b4ce-412a-9c11-0f82b8b61284)
